src/components/storePanel/StoreBanner/StoreBanner.module.scss

```diff
@@ -0,0 +1,87 @@
+@import '@/styles/index.scss';
+
+.banner {
+  display: flex;
+  flex-direction: column;
+  max-width: toRem(1008);
+  width: 100%;
+
+  @media (max-width: $tablet) {
+    width: calc(75% - 20px);
+  }
+
+  @media (max-width: $mobileSmall) {
+    width: 100%;
+  }
+
+  &_image {
+    & > * {
+      height: 180px;
+      width: 100%;
+      object-fit: cover;
+
+      @media (max-width: $mobileSmall) {
+        height: 124px;
+      }
+    }
+  }
+
+  &_inner_text {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+
+    & > p {
+      margin-right: 8px;
+      color: #333;
+      font-family: 'Craftwork Grotesk';
+      font-size: 14px;
+      font-style: normal;
```

Comment:
Don't just copy styles from a figma, think about what you're write and whether all the lines of code are necessary.